src/integrations/utils.ts

```diff
@@ -47,20 +47,46 @@ export async function initializeIntegrations(
   integs: any,
   featheryClient: Client
 ) {
+  // const isAuthIntegActive = Boolean(integs.stytch || integs.firebase);
   await Promise.all([
     installPlaid(!!integs.plaid),
     installFirebase(integs.fb),
     installStytch(integs.stytch),
```

Comment:
remove this from the general install. these auth integs should only be installed via the useAuth hook's preload